components/kubeflow/pytorch-launcher/component.yaml

```diff
@@ -0,0 +1,50 @@
+name: Kubeflow - Launch PyTorchJob
+description: Kubeflow PyTorchJob launcher
+inputs:
+- {name: name,                       type: String,                       description: 'PyTorchJob name.'}
+- {name: namespace,                  type: String,   default: kubeflow,  description: 'PyTorchJob namespace (likely your current namespace).'}
+- {name: version,                    type: String,   default: v1,        description: 'PyTorchJob version.'}
+- {name: master_spec,                type: Json,     default: '{}',      description: 'PyTorchJob Master replicaSpecs.'}
```

Comment:
Can you please replace `Json` type with the canonical `JsonObject` type name?
It's recognized by the SDK and the user can then pass dictionary as an argument which the SDK auto-converts to JSON.